pages/common/rtmpdump.md

```diff
@@ -0,0 +1,12 @@
+# rtmpdump
+
+> Small dumper for media content streamed over the RTMP protocol.
```

Comment:
I am not sure if adding the word "Small" helps here. Perhaps - A tool to dump media content streamed over the RTMP protocol ?